docs/UserGuide.md

```diff
@@ -68,26 +74,29 @@ Profile | me | category, descriptors
 
 **:information_source: Notes about the command format:**<br>
 
+* Commands that deal with Company, Internship, Application, and Profile can only be executed when you are on the appropriate tab in the app.
+The tab you must be on to execute a certain command is stated under each relevant section header in this guide.
+
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add -com n/COMPANY_NAME i/INDUSTRY... j/JOB_TITLE...`, `COMPANY_NAME`, `INDUSTRY` and `JOB_TITLE` are
-   parameters which can be used as `add -com n/Garena i/Gaming j/Game developer`.
+  e.g. in `add com n/COMPANY_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/INDUSTRY]...`, `COMPANY_NAME`, `PHONE_NUMBER`, `EMAIL`, `ADDRESS`, `INDUSTRY`,
+  are parameters which can be used as `add com n/Google p/65218000 e/GoogleHire@gmail.com a/70 Pasir Panjang Rd, #03-71 t/Cloud Computing t/Artificial Industry`.
 
 * Items in square brackets are optional.<br>
-  e.g `add -int n/COMPANY_NAME j/JOB_TITLE [i/INDUSTRY] [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...` can be used as <br/> `add -int n/Google j/Software Engineer i/Software
-`.
+  e.g `add int INDEX j/JOB_TITLE [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...` can be used as <br/> `add int 1 j/Software Engineer` or
+  `add int 3 j/Web Developer r/React w/3000 r/HTML5`.
 
 * Items with `...`​ after them can be used multiple times including zero times.<br>
   e.g. `[r/REQUIREMENT]...` can be used as `r/Rust`, `r/React native r/JavaScript` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `i/INDUSTRY r/REQUIREMENT`, `r/REQUIREMENT i/INDUSTRY` is also acceptable.
+  e.g. if the command specifies `p/PERIOD w/WAGE`, `w/WAGE p/PERIOD` is also acceptable.
 
-* `INDEX` refers to the index of the item in the displayed list of items (follows one-based indexing). <br>
+* `INDEX` refers to the index of the item (Company, Application, or Profile Item) in its respective displayed list of items (follows one-based indexing) unless stated otherwise. <br>
 
 
 ### Company
 
-Note: You must be on the **Companies** page in order to execute these commands.
+Note: You must be on the **Company** tab in order to execute these commands.
```

Comment:
I see we've changed the tab name from plural to singular. I was wondering if it would be odd since the user would be looking at a tab labelled "Company" but in it can be multiple companies? Likewise for "Application" as well, which to me sounds like I'll view a single application if I click on it. Small matter but perhaps we could discuss this further and see what the team thinks too!

Reply:
True I am not sure about which is a more appropriate wording.

Reply:
Alright, no worries! Will merge this first and we can settle this next time.